packages/storage/src/service.ts

```diff
@@ -42,15 +38,14 @@ import * as errorsExports from './implementation/error';
  */
 export class StorageService {
   private app_: FirebaseApp | null;
```

Comment:
This can be readonly as well.

Reply:
It seems like it can't because `deleteApp()` sets it to null.